### DISCUSSION TITLE
PyAnnote error

### DISCUSSION THREAD

**Discussion by CXhk87** [pyannote/pyannote-audio#1573] [Q&A]
Hi ! 
Thanks for NoScribe. I am new and do not succed to have my audio transcripted: and I do not understand why. This is the message I get. Can someone help me please ? 
![Capture d’écran 2023-11-30 à 22 48 18](https://github.com/pyannote/pyannote-audio/assets/152559385/2a31c6d8-bb4f-4c89-b62b-1f63ff957b54)

**Comment by hbredin** [comment]
Wrong repo? You should probably open an issue in `noScribe` repo. Closing.
